plugins/woocommerce/tests/e2e-pw/tests/merchant/create-checkout-block.spec.js

```diff
@@ -149,9 +149,11 @@ test.describe(
 						.locator( 'legend' )
 				).toBeVisible();
 				await expect(
-					page.locator(
-						'.wp-block-woocommerce-checkout-order-summary-block'
-					)
+					page
+						.locator(
+							'.wp-block-woocommerce-checkout-order-summary-block'
+						)
+						.first()
```

Comment:
On WPCOM, this one is resolved to more elements, so I'm making it more specific.